gamemodes/terrortown/gamemode/shared/sh_sprint.lua

```diff
@@ -22,13 +20,13 @@ SPRINT = {
 -- @return boolean
 -- @realm shared
 function SPRINT:PlayerWantsToSprint(ply)
-	local isSprinting = ply:KeyDown(IN_SPEED)
+	local inSprinting = ply:KeyDown(IN_SPEED)
```

Comment:
why? if you want to keep this in line with `inMovement` at least make it `inSprint`

Reply:
check the line below, it looked like a typo to me